packages/p2p-v2/src/components/AdvertiserName/AdvertiserName.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Avatar } from '../Avatar';
+import { useAdvertiserStats, useDevice } from '../../hooks';
+import AdvertiserNameStats from './AdvertiserNameStats';
+import AdvertiserNameBadges from './AdvertiserNameBadges';
+import AdvertiserNameToggle from './AdvertiserNameToggle';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import './AdvertiserName.scss';
+
+const AdvertiserName = () => {
+    const { data: advertiserStats, isLoading } = useAdvertiserStats();
+    const { isDesktop } = useDevice();
+
+    if (isLoading || !advertiserStats) return <h1>Loading...</h1>;
```

Comment:
shouldn't this be replaced with loader component

Reply:
Noted, I can fix it in the next PR I'm working on right now, cuz I need this profile content for constructing the profile page